### ISSUE TITLE
Include links to full lists of candidates

### ISSUE THREAD

**Issue by tdg** [swapmyvote/swapmyvote#606]
## Places where these links could be used

- Per constituency links when showing "willing" selection.  This would be nice since the linked info might lead users to other info (e.g. candidate's Twitter) which helps them decide who they are willing to vote for.
- Per candidate/party/constituency links on confirmed swap pages
- In FAQ: general links only like https://whocanivotefor.co.uk/

Relatively easy to do for by-elections, less so for General Elections!
 
https://whocanivotefor.co.uk/elections/parl.tiverton-and-honiton.by.2022-06-23/tiverton-and-honiton/
https://whocanivotefor.co.uk/elections/parl.wakefield.by.2022-06-23/wakefield/

## See also

- #348 
- #619.

**Comment by baob** [comment]
Yes, less easy to do a full solution for general elections. Seems you first have to search  democracy for elections that are uk parliament (the election id being parl.<date>) then find the individual ballot id which is specific to the constituency and from that you can construct the full url to the page. Example pages and links that might help here:

https://developers.democracyclub.org.uk/api/v1/

https://candidates.democracyclub.org.uk/

https://candidates.democracyclub.org.uk/media/candidates-parl.2022-06-23.csv

https://candidates.democracyclub.org.uk/elections/parl.aberavon.2019-12-12/

**Comment by baob** [comment]
Relates to https://github.com/swapmyvote/swapmyvote/issues/619

**Comment by baob** [comment]
If we provide a link to the candidates running in that constituency, it might invite the criticism that we're not offering swaps for all the parties on the linked candidate list. I think we'd need to
* Add covering text saying why all the parties aren't covered.
* or do something approximating to #619 first

**Comment by aspiers** [comment]
> If we provide a link to the candidates running in that constituency, it might invite the criticism that we're not offering swaps for all the parties on the linked candidate list.

Ah, I guess you mean that if we do this before resolving #2 and/or #619, then users following the link will see the full list of candidates and wonder why it's longer than the willing list we display for their constituency.  Good point.
